### PR TITLE
conflict when using picker popOver on a popOver contains the form

### PR DESCRIPTION
**Pull request by e3bmo3ty**
when using the form on iPad contained in popOver, when click on picker the main popOver hide and picker popOver shows.

solved by a flag  "forceIphoneBehavior" passed to the field parameter so i can prevent the picker to show in a popOver

### CODE REVIEW

**Comment by FokkeZB** [comment]
Thanks! Could you change the arg to be `disablePopover` to be more specific in what it does rather then what it emulates?

**Comment by e3bmo3ty** [comment]
Done :)

File: controllers/picker.js [hunk 1]
```diff
@@ -70,9 +71,11 @@ var picker = {
...
 
   // compose view
-  if (Ti.Platform.osname === 'ipad') {
-    $.win.title = $.label.text;
-    $.win.add($.picker);
+  if (Ti.Platform.osname === 'ipad' && !args.disablePopover) {
+      $.win.title = $.label.text;
+      $.win.add($.picker);
+  } else {
+      forceIphoneBehavior = args.disablePopover;
```
Comment (FokkeZB) [inline controllers/picker.js]: Should be `disablePopover`

File: controllers/picker.js [hunk 2]
```diff
@@ -70,9 +71,11 @@ var picker = {
   $.setInput($.input);
 
   // compose view
-  if (Ti.Platform.osname === 'ipad') {
-    $.win.title = $.label.text;
-    $.win.add($.picker);
+  if (Ti.Platform.osname === 'ipad' && !args.disablePopover) {
```
Comment (FokkeZB) [inline controllers/picker.js]: Please also document the new arg in the JSdoc for the function

**Comment by FokkeZB** [comment]
Hi @e3bmo3ty, I see you closed the PR? Why so?

**Comment by FokkeZB** [comment]
Ah, I see now you've opened #33 to replace this. Thanks!

**Pull request closed without merging**